modules/renter/proto/negotiate.go

```diff
@@ -12,6 +12,14 @@ import (
 	"github.com/NebulousLabs/Sia/types"
 )
 
+type payoutAdjustmentInput struct {
+	outputs    []types.SiacoinOutput
+	numOutputs uint
```

Comment:
should be able to determine `numOutputs` by calling `len(outputs)`, if those aren't the same we've got other problems I think